src/AlgoliaSearch.js

```diff
@@ -118,54 +119,56 @@ AlgoliaSearch.prototype = {
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  deleteIndex: function(indexName, callback) {
-    return this._jsonRequest({ method: 'DELETE',
-              url: '/1/indexes/' + encodeURIComponent(indexName),
-              hostType: 'write',
-              callback: callback });
+  deleteIndex: function deleteIndex(indexName, callback) {
```

Comment:
This could be a rule to enforce (force naming of the methods) http://eslint.org/docs/rules/func-names ?